packages/application-components/package.json

```diff
@@ -42,22 +42,22 @@
     "@commercetools-frontend/assets": "13.8.0",
     "@commercetools-frontend/constants": "13.9.0",
     "@emotion/core": "10.0.10",
-    "@emotion/styled": "10.0.11",
+    "@emotion/styled": "10.0.12",
     "prop-types": "15.7.2",
     "react-modal": "3.8.1",
     "warning": "4.0.3"
   },
   "devDependencies": {
     "@commercetools-frontend/ui-kit": "9.8.3",
-    "@storybook/addon-actions": "5.1.1",
+    "@storybook/addon-actions": "5.1.3",
     "@storybook/addon-contexts": "5.1.3",
-    "@storybook/addon-info": "5.1.1",
-    "@storybook/addon-knobs": "5.1.1",
-    "@storybook/addon-links": "5.1.1",
-    "@storybook/addon-options": "5.1.1",
-    "@storybook/addon-storysource": "5.1.1",
-    "@storybook/react": "5.1.1",
-    "@storybook/theming": "5.1.1",
+    "@storybook/addon-info": "5.1.3",
```

Comment:
it's SO WEIRD that it's working now 🤔 

I spent a few hours trying to get the latest storybook on app kit working with core js